paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/CardUiDefinitionFactoryTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.stripe.android.lpmfoundations.paymentmethod.definitions
+
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadataFactory
+import com.stripe.android.model.PaymentIntentFixtures
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.addresselement.AddressDetails
+import com.stripe.android.screenshottesting.PaparazziRule
+import org.junit.Rule
+import org.junit.Test
+
+class CardUiDefinitionFactoryTest {
+    @get:Rule
+    val paparazziRule = PaparazziRule()
```

Comment:
Not testing any permutations, just the default. Since we're mostly testing layout and presence here, and the individual fields can test all permutations.